addons/account_edi_ubl_cii/models/account_move_send.py

```diff
@@ -124,6 +124,10 @@ def _hook_invoice_document_after_pdf_report_render(self, invoice, invoice_data):
         # EXTENDS 'account'
         super()._hook_invoice_document_after_pdf_report_render(invoice, invoice_data)
 
+        # Skip proforma pdf post processing
```

Comment:
I can see that in further versions, we use `proforma_pdf_attachment_values` when reading the pdf content below
in 17.0:
```python
pdf_values = invoice_data.get('pdf_attachment_values') or invoice_data['proforma_pdf_attachment_values']
reader_buffer = io.BytesIO(pdf_values['raw'])
 ```

so I don't think this would be the right solution here :thinking: